Modify field Trial flow chart and edits to RC2 Chptr 4 & 5 (#3137)
cntt-n/CNTT: doc/gov/chapters/chapter09.rst

   -  Community (Anuket)
   -  LaaS (e.g. UNH)

> 3. Engaged vendors successfully configure their VI and run the RC-1 or RC-2 test suite and are able to provide expert feedback

Reviewer: ```suggestion
3. Engaged vendors successfully configure their VI and run the RC1 or RC2 test suite and are able to provide expert feedback
```

Author: Will remove the dashes.

   * Community (Anuket)
   * LaaS (e.g. UNH)

> #. Engaged vendors successfully configure their VI and run the RC1 or RC2 test suite and are able to provide expert feedback

Reviewer: ```suggestion
#. Engaged vendors successfully configure their Cloud Infrastructure and run the RC1 or RC2 test suite and are able to provide expert feedback
```

Objectives of RI1/RC1 Trials
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The object is to quantitively and qualitatively assess and evaluate the following Anuket requirements,
methods, and support processes:

> -  RI1 or RI2 Specifications

Reviewer: ```suggestion
-  RA1 or RA2 Specifications
```

> -  VI implementation support methods ( i.e. cookbooks, installation manuals, how to guides etc.)

Reviewer: ```suggestion
-  Cloud Infrastructure implementation support methods ( i.e. cookbooks, installation manuals, how to guides etc.)
```

Required Assessments
^^^^^^^^^^^^^^^^^^^^

> -  VI Implementation methods and procedures (cookbook, etc)

Reviewer: ```suggestion
-  Cloud Infrastructure Implementation methods and procedures (cookbook, etc)
```

> -  RI1 or RI2 Specifications

Reviewer: ```suggestion
-  RA1 or RA2 Specifications
```
